tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "moduleResolution": "node",
     "experimentalDecorators": true,
     "noUnusedLocals": true,
+    "strictNullChecks": false,
+    "noImplicitAny": false,
```

Comment:
I disagree with both of these. I’d rather add some extra code to make sure the values are correct than deal with false negatives in production. If necessary, I’d rather resort to `// @ts-ignore` for a problematic line of code and add an explanatory comment.

Reply:
Good point. I've reverted tsconfig back, and weirdly, not getting any Implicit any warnings anymore when I build. Guess that was a bit of compiler panic...